docs/helm-chart/production-guide.rst

```diff
@@ -184,6 +184,44 @@ To use an external StatsD instance:
       statsd_host: ...
       statsd_port: ...
 
+Datadog
+^^^^^^^
+If you are using a Datadog agent in your environment, this will enable Airflow to export metrics to the Datadog agent.
+NOTE: The below is for version 2.x.x of Airflow.  In Airflow 1.x.x the swap all instances of ``metrics`` for ``scheduler``
+
+.. code-block:: yaml
+
+  statsd:
+    enabled: false
+  config:
+    metrics:
+      statsd_on: true
+      statsd_port: ``<port>``
```

Comment:
```suggestion
      statsd_port: 8125
```

I think we should show the default dogstastsd port, thoughts?

Reply:
I'm fine with that. May want to add a comment to suggest that 8125 is default for dogstatsd but may differ in deployed environment or something.